### PR TITLE
Using the new shared include directory files in the FPGA merge_sort design.

### PR DESCRIPTION
**Pull request by tyoungsc**
# Existing Sample Changes
## Description
Deleting files from the FPGA merge_sort design in favour of the new shared include files.

## Type of change
- [X] code cleanup

## How Has This Been Tested?
- [X] Command Line
- [X] regtest

### CODE REVIEW

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/main.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
 #include "dpc_common.hpp"
 
-// include the merge sort kernel header
+#include "constexpr_math.hpp"
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/main.cpp:17]: I like that you commented in merge_sort.hpp where the shared files can be found, I suggest we always do this.  When it's just a single file you can add the comment on the same line as the #include.

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge.hpp [hunk 2]
```diff
@@ -4,8 +4,8 @@
 #include <CL/sycl.hpp>
 #include <sycl/ext/intel/fpga_extensions.hpp>
 
+#include "constexpr_math.hpp"
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge.hpp:7]: Add comment for location of this file

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp [hunk 3]
```diff
@@ -14,9 +14,11 @@
 #include "merge.hpp"
 #include "produce.hpp"
 #include "sorting_networks.hpp"
+
+// Included from DirectProgramming/DPC++FPGA/include/
+#include "pipe_utils.hpp"
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp:19]: Include files should be in alphabetical order

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp [hunk 4]
```diff
@@ -163,7 +165,7 @@ std::vector<event> SubmitMergeSort(queue& q, size_t count, ValueT* buf_0,
   ValueT* buf[2] = {buf_0, buf_1};
 
   // using double buffering, so track the current buffer and have a simple
-  // lamda to compute the next buffer index
+  // lambda to compute the next buffer index
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp:168]: I'm embarassed that I reviewed so much of your code without realizing that you don't know how to spell 'lambda' :-(

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/sorting_networks.hpp [hunk 5]
```diff
@@ -6,7 +6,7 @@
 #include <CL/sycl.hpp>
 #include <sycl/ext/intel/fpga_extensions.hpp>
 
-#include "impu_math.hpp"
+#include "constexpr_math.hpp"
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/sorting_networks.hpp:9]: comment for file location

**Comment by mdbtucker** [comment]
Do you need to update the README file to reflect the new location of the shared include file(s)?

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp (hunk 3 above)
Comment (tyoungsc) [reply DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/merge_sort.hpp:19]: Not nice for batching things under '// Included from DirectProgramming/DPC++FPGA/include/'

**Review by mdbtucker — APPROVED** [review]

**Review by mdbtucker — APPROVED** [review]

**Review by mdbtucker — APPROVED** [review]

**Pull request merged**